requirements.txt

```diff
@@ -1,3 +1,4 @@
+flask
```

Comment:
Is it possible to avoid installing an extra web server? We already use Tornado web server,  could you redo this in Tornado to make our ParlAI installation leaner? Thanks!

Reply:
@mojtaba-komeili Thank you for the guidance, I have created a tornado script with Parlai setup_args but not help with the result still trying to improve it, I'll get back to you once it will be completed and any idea about the flask solution? @stephenroller

Reply:
Flask is fine by me, and we don't need this imported for `parlai flask_demo`, at least not now.